.coderabbit.yaml

```
@@ -13,24 +13,32 @@ reviews:
+    gitleaks:
+      enabled: true
+      scan_secrets: true
+    osvScanner:
+      enabled: true
+      scan_dependencies: true
+      vulnerability_alerts: true
+    eslint:
+      enabled: true
 
+  # Supported review tools
+  tools:
```

YAMLlint (1.37.1)

[error] duplication of key "tools" in mapping

(key-duplicates)